src/frontend/apps/desk/src/features/mail-domains/domains/types.ts

```diff
@@ -21,3 +22,5 @@ export enum Role {
   OWNER = 'owner',
   VIEWER = 'viewer',
 }
+
+export const ORDERED_ROLES = [Role.VIEWER, Role.ADMIN, Role.OWNER];
```

Comment:
I have the feeling this constant will be only necessary in the component `ChooseRole`, what about scoping it in the component itself ?